usecase/command/evidence.go

```diff
@@ -0,0 +1,35 @@
+package command
+
+import (
+	entity_event "github.com/crypto-com/chain-indexing/entity/event"
+	"github.com/crypto-com/chain-indexing/usecase/event"
+	"github.com/crypto-com/chain-indexing/usecase/model"
+)
+
+type Evidence struct {
+	blockHeight int64
+	params      model.EvidenceParams
+}
+
+func NewEvidence(blockHeight int64, params model.EvidenceParams) *Evidence {
+	return &Evidence{
+		blockHeight,
+		params,
+	}
+}
+
+// Name returns name of command
+func (*Evidence) Name() string {
+	return "Evidence"
```

Comment:
I think we are lacking the very here, for example `SubmitEvidence` or `CreateEvidence` etc

Reply:
Yeah, thanks! Good catch. WIll change it to `EvidenceSubmitted`.